apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/RifLayoutsProcessor.java

```diff
@@ -1503,4 +1828,195 @@ private void writeDebugLogMessages() {
+
+  /**
+   * Creates the fields for the BeneficiaryMonthly class in the model rif
+   *
+   * @param lineEntity helps build the entity {@link TypeSpec.Builder}
+   * @param isId determines if the field is an id field
+   * @param isTransient determines if the field is transient {@link boolean}
+   * @param isColumnOptional determines if the field is optional {@link boolean}
+   * @param fieldName specifies the fieldname {@link String}
+   * @param type specifies the field type {@link RifColumnType}
+   * @param columnLength specifies the column length {@link Optional<Integer>}
+   * @param columnScale specifies the column scale {@link Optional<Integer>}
+   */
+  private static void createBeneficiaryMonthlyFields(
+      TypeSpec.Builder lineEntity,
+      boolean isId,
+      boolean isTransient,
+      boolean isColumnOptional,
+      String fieldName,
+      RifColumnType type,
+      Optional<Integer> columnLength,
+      Optional<Integer> columnScale) {
+    FieldSpec lineField =
+        FieldSpec.builder(
+                selectJavaFieldType(type, isColumnOptional, columnLength, columnScale),
+                fieldName,
+                Modifier.PRIVATE)
+            .addAnnotations(
+                createBeneficiaryMonthlyAnnotations(
+                    isId,
+                    isTransient,
+                    isColumnOptional,
+                    fieldName,
+                    type,
+                    columnLength,
+                    columnScale))
+            .build();
+    lineEntity.addField(lineField);
+
+    MethodSpec.Builder lineFieldGetter =
+        MethodSpec.methodBuilder(calculateGetterName(lineField))
+            .addModifiers(Modifier.PUBLIC)
+            .returns(selectJavaPropertyType(type, isColumnOptional, columnLength, columnScale));
+    addGetterStatement(isColumnOptional, lineField, lineFieldGetter);
+    lineEntity.addMethod(lineFieldGetter.build());
+
+    MethodSpec.Builder lineFieldSetter =
+        MethodSpec.methodBuilder(calculateSetterName(lineField))
+            .addModifiers(Modifier.PUBLIC)
+            .returns(void.class)
+            .addParameter(
+                selectJavaPropertyType(type, isColumnOptional, columnLength, columnScale),
+                lineField.name);
+    addSetterStatement(isColumnOptional, lineField, lineFieldSetter);
+    lineEntity.addMethod(lineFieldSetter.build());
+  }
+
+  /**
+   * Creates the fields for the BeneficiaryMonthly annotations in the model rif
+   *
+   * @param isId determines if the field is an id field
+   * @param isTransient determines if the field is transient {@link boolean}
+   * @param isColumnOptional determines if the field is optional {@link boolean}
+   * @param fieldName specifies the fieldname {@link String}
+   * @param type specifies the field type {@link RifColumnType}
+   * @param columnLength specifies the column length {@link Optional<Integer>}
+   * @param columnScale specifies the column scale {@link Optional<Integer>}
```

Comment:
Would you mind applying this suggestion?
```suggestion
   * @param columnLength specifies the column length {@link Optional<Integer>}, for numeric types this represents the total number of digits that can be stored
   * @param columnScale specifies the column scale {@link Optional<Integer>}, for numeric types this represents how many of the total digits (see `columnLength`) are to the right of the decimal point
```